readme.md

```diff
@@ -5,7 +5,8 @@ Safe and fast serialization alternative to [JSON.stringify][].
 Gracefully handles circular structures instead of throwing.
 
 Provides a deterministic ("stable") version as well that will also gracefully
-handle circular structures. See the example below for further information.
+handle circular structures. See the example below for further information. For
+the stable version also look at: [`safe-stable-stringify`][].
```

Comment:
I think remove this in favour of mentioning it in the context of stable stringify method